Source/santad/SNTSyncdQueue.mm

```diff
@@ -15,23 +15,28 @@
 
 #import "Source/santad/SNTSyncdQueue.h"
 
+#include <memory>
+
 #import "Source/common/MOLXPCConnection.h"
 #import "Source/common/SNTLogging.h"
 #import "Source/common/SNTStoredEvent.h"
 #import "Source/common/SNTXPCSyncServiceInterface.h"
+#include "Source/common/SantaCache.h"
+#include "Source/common/String.h"
 
 @interface SNTSyncdQueue ()
-@property NSCache<NSString *, NSDate *> *uploadBackoff;
 @property dispatch_queue_t syncdQueue;
 @end
 
-@implementation SNTSyncdQueue
+@implementation SNTSyncdQueue {
+  // TODO: Eventually replace with an LRU.
```

Comment:
nit: we should open an issue to track this.

Reply:
Good idea, done.

Reply:
saw https://github.com/northpolesec/santa/issues/344 thanks